.changeset/curvy-houses-remember.md

```diff
@@ -0,0 +1,48 @@
+---
+'@nhost/react': major
+---
+
+Remove deprecated signatures in React hooks
```

Comment:
Our changelogs should not be this long. Turn this into a PR description instead and use a short description.